content/docs/faq-build.md

```diff
@@ -1,32 +1,32 @@
 ---
 id: faq-build
-title: Babel, JSX, and Build Steps
+title: Babel و JSX و خطوات بناء التطبيقات
```

Comment:
```suggestion
title: Babel و JSX وخطوات بناء التطبيقات
```

Reply:
Is it possible to clarify more !!

Reply:
I think he's just suggesting that we remove an unnecessary whitespace 😅

Reply:
are you serious 🤣  !

Reply:
@3imed-jaberi Yes, as Mohammad said, the whitespace is unnecessary, sorry I should mention it.
This is a common mistake and Hsoub's writing guidelines also mention it, I just can't find the link where I read it.
But you can check this [link](https://www.bbc.co.uk/academy/ar/articles/art20130702112134021) as well.